MAAP CMR and .ops updates

smk0033 opened the pull request:
Tutorials so far that work and can be updated to remove .ops and/or MAAP CMR. Technical and science tutorials - pushing technical first, science will follow. Github ticket for reference: https://github.com/orgs/NASA-IMPACT/projects/29?pane=issue&itemId=27143805

Other technical tutorial issues are listed [here](https://github.com/NASA-IMPACT/active-maap-sprint/issues/597)
Other science tutorial updates listed [here](https://github.com/NASA-IMPACT/active-maap-sprint/issues/554)

smk0033 commented:
Note about HLS tutorial: seems to work with the .ops update, but it does like to crash on me when it pulls all of the different bands (I think it even did this before I removed it). Made note of it in the individual issue for HLS when we do VEDA updates, but I'm going to go ahead and push it here

smk0033 commented:
I'll leave a comment when I finish updating the science tutorials with any more cmr/ops changes that I can make, which should be tomorrow afternoon. After that, I'll start looking and making other edits to each individual notebook and do separate PRs

emileten commented on docs/source/technical_tutorials/access/accessing_gedi_l4a_data.ipynb:
```diff
@@ -2,20 +2,20 @@
  "cells": [
```
<p>I tried to click on the 'Searching for Granules in MAAP' https://docs.maap-project.org/en/latest/search/granules.html but I get a 404 not found</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

emileten replied:
@smk0033

jjfrench replied:
<p>I believe this is what it should point to https://docs.maap-project.org/en/latest/technical_tutorials/search/granules.html</p><p>Does that look right?</p>

smk0033 replied:
Yup! I think I have it updated to that in a new PR for the accessing_gedi_l3 tutorial, and I'll have it updated to that as well soon for the gedi_l4a tutorial. I wasn't sure if I should start pushing individual tutorials until we get this PR merged

emileten commented:
Was it ready for review ? I did a quick review @smk0033, apart from the above comment it looks ok to me.

In addition, could you reference the issue this PR is solving ? My guess is that it's https://github.com/orgs/NASA-IMPACT/projects/29?pane=issue&itemId=27143805 but I am not sure. Thanks !

smk0033 commented:
Sure thing! I'll have the rest of the science tutorials that can be pushed soon, but thanks for looking at what I've gotten so far @emileten! I also plan to fix the links when we go through the individual tutorials after this PR, my biggest thing with this ticket was fixing cmr/.ops in the tutorials that allowed it (and that don't seem to have anything else broken besides links)

smk0033 commented:
I think that's all of the science tutorials I can go ahead and update just for this ticket. Other issues have been noted in the individual tickets under VEDA guidelines

smk0033 commented on docs/source/science/HLS/HLSL30.ipynb:
```diff
@@ -2,23 +2,23 @@
  "cells": [
```
<p>This was! I tried to upgrade the memory earlier so I'll see if it runs now</p>

smk0033 commented:
I think I've fixed all the links in the tutorials provided so far for this PR (please let me know if I missed any!). I also added searching_compiling_list_of_granule_ids on here, it didn't have any cmr/ops changes, but I accidentally updated it and just went with it since it was only a link and text fix. I was also able to rerun the HLS tutorial and it seemed like it processed with no issues

jjfrench reviewed (approved):
All listed notebook links and references seem to be updated 👍🏼

emileten reviewed (approved):
Thanks for the link and referencing the ticket @smk0033 ! LGTM
